### PR TITLE
Fix/mobile layout

### LINKED ISSUE #49: Fix mobile layout bugs

### PR DESCRIPTION
**Pull request by danbull**
Fixes #49 

- Replaced bootsrap with Gravity
- Minor type weight changes
- New Team styling on mobile

### CODE REVIEW

File: src/styles/_contact.scss [hunk 1]
```diff
@@ -8,6 +8,12 @@
     padding: 5px 10px;
   }
 
+  @media (min-width: grav-breakpoint(small)) {
+    h2 + div {
+      max-width: 83.33333%;
```
Comment (patrickbower) [inline src/styles/_contact.scss]: whats this ... it seems a bit sketchy although I'm not sure what's this is doing? Could flex-box be a better option?

File: src/styles/_gravity-overrides.scss [hunk 2]
```diff
@@ -7,23 +7,35 @@ main {
   margin-top: 0;
 }
 
+img {
+  width: 100%;
```
Comment (patrickbower) [inline src/styles/_gravity-overrides.scss]: isn't there a max-width already present and do we even have any images (there all SVGs no?)

Comment (danbull) [reply src/styles/_gravity-overrides.scss]: This was to fix an issue with img's but yes no longer needed now

File: src/styles/_contact.scss (hunk 1 above)
Comment (danbull) [reply src/styles/_contact.scss]: Just copied the width that bootstrap was setting

**Pull request merged**